src/test/java/api/support/examples/LocationExamples.java

```diff
@@ -11,19 +12,25 @@
   private final UUID jubileeCampusId;
   private final UUID businessLibraryId;
   private final UUID primaryServicePointId;
+  private final UUID secondaryServicePointId;
+  private final UUID tertiaryServicePointId;
 
   public LocationExamples(
     UUID nottinghamUniversityId,
     UUID jubileeCampusId,
     UUID businessLibraryId,
     UUID djanoglyLibraryId,
-    UUID primaryServicePointId) {
+    UUID primaryServicePointId,
+    UUID secondaryServicePointId,
```

Comment:
The domain does not have the concept of secondary or tertiary service points, only a single primary for a location and any other that serve that location.

Could the final two parameters be replaced by a variable argument?

Reply:
Yes, will change